catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/transform/QuerySortsValueTransformer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.transform;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.impl.AttributeImpl;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.codice.ddf.catalog.ui.metacard.workspace.QueryMetacardTypeImpl;
+
+public class QuerySortsValueTransformer implements QueryValueTransformer {
+
+  @Override
+  public boolean filter(String key, Object value) {
+    return QueryMetacardTypeImpl.QUERY_SORTS.equals(key);
+  }
+
+  @Override
+  public Attribute createAttribute(String key, Object value) {
+    List<Map<Serializable, Serializable>> values = jsonValueToMetacardValue((List<?>) value);
+
+    return new AttributeImpl(key, (Serializable) values);
+  }
+
+  private <T extends Map<Serializable, Serializable>> List<T> jsonValueToMetacardValue(
+      List<?> jsonValue) {
+    return (List<T>)
```

Comment:
❓Isn't `QUERY_SORTS` a list of strings? I thought this code should look similar to this: https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/impl/QuerySortsParser.java#L45

Reply:
@bellcc Has this been resolved?

Reply:
Just addressed this. Made two changes with this:
1. Value transformers now have a method for transforming their stored value to a JSON value. 
2. The sorts field was changed back to the old way of storing things.